lib/responsys/api/client.rb

```diff
@@ -9,8 +10,17 @@ class << self
         alias :instance :new
       end
 
+      def initialize
+        @raise_exceptions = false
+      end
+
       def api_method(action, message = nil, response_type = :hash)
-        raise Responsys::Helper.get_message("api.client.api_method.wrong_action_#{action.to_s}") if action.to_sym == :login || action.to_sym == :logout
+        raise GenericException.new("api.client.api_method.wrong_action_#{action.to_s}") if action.to_sym == :login || action.to_sym == :logout
```

Comment:
should this be `ParameterException` since it's complaining about the action value provided the method? :nitpicky: